### ISSUE TITLE
"Unhandled exception" occurring

### ISSUE THREAD

**Issue by Asterra** [PeterTh/gedosato#50]
I get this when clicking "alternative injection" or "start with Windows", or indeed "default injection" assuming I ignore the error.  In-game (Divinity: Original Sin) there is no change under any circumstance, presumably as the alternative injection is reportedly required, and/or some facet of this bug is preventing the injection from working as a whole.  Running as admin with UAC disabled.

Win7 64, R9 290X, i7-980X.

Whole text of error:
"Unhandled exception has occurred in your application.  If you click Continue, the application will ignore this error and attempt to continue.  If you click Quit, the application will close immediately.

Attempted to perform an unauthorized operation."

**Comment by hasantareq** [comment]
I'm having the same problem as Asterra. And, it happens for the latest version 'Chaos Flare'.

Well, I copied the downloaded files from my laptop to my desktop with OS win 8.1 64-bit. Previously, I used the same version of gedosato with win 7 64-bit & gedosato worked perfectly for games with final fantasy xiii, dark souls ii.

I have always ran gedosato as administrator. Updated all of the Visual C++ 2015, 2013, 2012, 2008 & also the DOT NET FX. But, the problem persist. And the program just doesn't hook to any games at all. And, YES the UEFI Secure booting was turned off all the time.

My windows 8.1 is a clean & legit install & my Desktop PC specs are -
   Pro    - Intel Core i7 4.0 gHz SKYLAKE EDITION
   RAM  -  32 gb
   gpu   -  ATI R9 390X 
   SSD - 160 gb
   HDD - 5 TB
The games (which previously worked on win 7 64-bit) that I have tried with are -
  1. Final Fantasy XIII & XIII-2
  2. Lightning Returns: Final Fantasy XIII
  3. Dark Souls II
  4. Dmc: Devil May Cry
  5. Devil May Cry 4: SE
I am also adding a screenshot.
![1](https://cloud.githubusercontent.com/assets/16418862/11985932/fd00ab7c-a9f3-11e5-90d2-147ac217fc08.jpg)
